src/components/cast/_cast.scss

```diff
@@ -8,12 +8,22 @@
     background-color: rgba(255, 255, 255, 0);
     border: none;
     cursor: pointer;
-    --connected-color: rgba(1, 172, 205, 0.8);
+    --connected-color: #{$grayscale4};
     --disconnected-color: #{$grayscale4};
+
+    &:hover {
+      --connected-color: #{$white};
+      --disconnected-color: #{$white};
+    }
   }
 
-  .cast-button:hover {
-    --disconnected-color: #{$white};
-    --connected-color: #{$brand-color};
+  .cast-button.cast-button-active {
```

Comment:
can move this to the `.player` block? and just use `.cast-button-active` ?

Reply:
I can use &.cast-button-active inside the cast-button block, is that what you mean ?
wanted to separate because it seemed more readable but not critical

Reply:
ok